pyspecdata/core.py

```diff
@@ -3538,32 +3558,67 @@ def dot(self,arg):
         >>> b = nddata(r_[0:9],[3,3],['a','d'])
         >>> print a.C.dot(b)
         >>> print tensordot(a.data,b.data,axes=((0),(0)))
+
+        .. literalinclude:: ../examples/matrix_mult.py
```

Comment:
if you build the sphinx, does this show up correctly?

Reply:
not sure if this pertains to nnls or tikhonov